packages/css-library/src/stylesheets/_mixins.scss

```diff
@@ -49,9 +49,8 @@ $default-feature: min-width;
   }
 }
 
-$color-gray-light:#aeb0b5;
-$focus-outline: 2px dotted $color-gray-light;
-$focus-spacing: 3px;
+$focus-outline: 2px solid $vads-color-action-focus-on-light;
+$focus-spacing: 2px;
```

Comment:
This fixes focus outline for anchor tags that are not contained in one of our web components. A lot of these anchor tags tend to live on hub pages somewhere. This mirrors the [web components mixin](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/mixins/focusable.css). 

Eventually, we'll want to probably have css-library be the source of truth for mixins, and instead inherit web-component mixins from there.